README.md

```diff
@@ -2,10 +2,31 @@
 
 # State management in iOS
 
+## Introduction
+
+In iOS development it's always good to choose a right architecture.
+As you know in projects used `MVC` pattern it becomes hard
+to maintain and test massive view controller after some time.
+
+Let's try to make it more clean and reusable using one way data flow architecture.
```

Comment:
Maybe change "one way data flow architecture" to "unidirectional data flow pattern" as you mention it below.